`mirte_msgs` build failing due to CMake `CMP0094`

Several packages have their CMake minimum version set to `3.8` in the `CMakeLists.txt`.

Example:

https://github.com/mirte-robot/mirte-ros-packages/blob/78f1033c62297497dfe2657e497f1738cf5e056e/mirte_msgs/CMakeLists.txt#L1-L2

this appears to be problematic on systems with more recent CMake releases which try to use non-global Python installs (such as when Conda is active), as [CMP0094](https://cmake.org/cmake/help/latest/policy/CMP0094.html) changed how CMake tries to find the interpreter and other parts of Python.

Example error:

```
CMake Error at /.../lib/python3.12/site-packages/cmake/data/CMake.app/Contents/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python (missing: Python_EXECUTABLE Python_INCLUDE_DIRS
  Python_LIBRARIES Python_NumPy_INCLUDE_DIRS Interpreter Development NumPy
  Development.Module Development.Embed)
```

---

Edit: @Chris-Pek FYI

--- a/mirte_msgs/CMakeLists.txt
+++ b/mirte_msgs/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.15)
 project(mirte_msgs)
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "clang")

